conf/portal.application.conf

```diff
@@ -272,14 +331,46 @@ akka {
   logging-filter = "akka.event.slf4j.Slf4jLoggingFilter"
 
   actor {
+    provider="akka.cluster.ClusterActorRefProvider"
+    serializers {
+      proto="akka.remote.serialization.ProtobufSerializer"
+    }
     debug {
       unhandled = on
     }
-    serializers {
-      proto="akka.remote.serialization.ProtobufSerializer"
+  }
+  cluster {
+    seed-nodes=["akka.tcp://mportal@127.0.0.1:2558"]
+    auto-down-unreachable-after = 300s
+    roles = ["host_indexer", "rollup_metrics_discovery"]
+    sharding {
+      guardian-name="sharding"
+      role=""
+      retry-interval="2 s"
+      buffer-size=100000
+      handoff-timeout="60 s"
+      rebalance-interval="10 s"
+      snapshot-interval="720 s"
+      state-store-mode="persistence"
+      least-shard-allocation-strategy {
+        rebalance-threshold=10
+        max-simultaneous-rebalance=3
+      }
     }
-    serialization-bindings {
```

Comment:
Any particular reason you dropped this?

Reply:
I don't think it's necessary, is it?

Reply:
In that case, the serializers config shouldn't be required as well.  Please remove it